src/components/NcCheckboxRadioSwitch/NcCheckboxRadioSwitch.vue

```diff
@@ -298,12 +313,19 @@ export default {
 			:button-variant="buttonVariant"
 			:is-checked="isChecked"
 			:loading="loading"
+			:description="description"
+			:label-id="labelId"
+			:description-id="descriptionId"
 			:icon-size
 			@click="onToggle">
 			<template #icon>
 				<!-- @slot The checkbox/radio icon, you can use it for adding an icon to the button variant -->
 				<slot name="icon" />
 			</template>
+			<template v-if="$slots.description" #description>
+				<!-- @slot The checkbox/radio description, you can use it for adding an complex description element to the button variant as opposed to the description slot -->
```

Comment:
This description makes no sense, no? Because this is about the description slot.
Also for button variant it does not exist?